src/GitHub.Api/Application/ApplicationManagerBase.cs

```diff
@@ -72,9 +72,22 @@ public void Run(bool firstRun)
                         }
                     });
 
+                var archiveParentPath = NPath.CreateTempDirectory("git_zip_paths");
+                var gitArchiveFilePath = AssemblyResources.ToFile(ResourceType.Platform, "git.zip", archiveParentPath, Environment);
+                if (!gitArchiveFilePath.FileExists())
+                {
+                    gitArchiveFilePath = null;
+                }
+
+                var gitLfsArchivePath = AssemblyResources.ToFile(ResourceType.Platform, "git-lfs.zip", archiveParentPath, Environment);
+                if (!gitLfsArchivePath.FileExists())
```

Comment:
This feels a bit better.. Maybe a static method in `GitInstaller` would be better than this...

Reply:
Ehh...